src/utils/urls.js

```diff
@@ -1,5 +1,12 @@
+function check() {
+  if (process.env.REACT_APP_LOCAL_ENV === 'true') {
+    return 'http://' + window.location.hostname + ':4000';
+  }
+  return 'https://api.susi.ai';
+}
+
 const urls = {
-  API_URL: process.env.REACT_APP_API_URL || 'https://api.susi.ai',
+  API_URL: check(),
   SOUND_SERVER_API_URL: 'http://0.0.0.0:7070',
```

Comment:
Why not use the same `'http://' + window.location.hostname + ':7070'; here?`

Reply:
@norbusan do you mean `SOUND_SERVER_API_URL`, that doesn't need to be present at all as all endpoints on the control server is relative to this page,I will make an issue for it.

Reply:
Which issue did you open?

Reply:
@mariobehling i opened it just now - https://github.com/fossasia/susi.ai/issues/2806